interfaces/Libraries.ts

```diff
@@ -81,6 +82,23 @@ const adapter_transformers = (model: ModelData) =>
 model = ${model.config.adapter_transformers.model_class}.from_pretrained("${model.config.adapter_transformers.model_name}")
 model.load_adapter("${model.modelId}", source="hf")`;
 
+const allennlpUnknown = () =>
+`unknown model type `
+
+const allennlpQuestionAnswering = (model: ModelData) =>
+`import allennlp_models
+from allennlp.predictors.predictor import Predictor
+
+predictor = Predictor.from_path("${model.modelId}")
```

Comment:
The model "path" should have "hf://" at the beginning, otherwise `Predictor` will treat it as a local path. Was that the intention? If not, here's a PR with the change: https://github.com/huggingface/huggingface_hub/pull/80

Reply:
Thanks for the quick follow-up PR!! The snippet was done based on the current Inference API implementation, hence leading to this bug.

Reply:
My pleasure!